tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/test_settings_change_language.py

```diff
@@ -10,17 +10,20 @@
 
 class TestChangeLanguage(GaiaTestCase):
 
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+        self.data_layer.set_setting('devtools.pseudolocalization.enabled', True)
+
     def test_change_language_settings(self):
         lang_name = self.marionette.execute_script("""
-            var  = window.wrappedJSObject.navigator.mozL10n.;
-            return ['fr-x-psaccent'].translate('Packaged Accented');
+            var qps = window.wrappedJSObject.navigator.mozL10n.qps;
```

Comment:
I don't really know, but that was what was accidentally removed in:
https://github.com/mozilla-b2g/gaia/commit/f03380e451eb2a33a27e7b30c183c311f8e7430d#diff-9413f92730386581ed79f16a649180fa
(from Bug 1217733)